src/main/java/net/mcreator/ui/modgui/StructureGUI.java

```diff
@@ -83,6 +83,7 @@ public StructureGUI(MCreator mcreator, ModElement modElement, boolean editingMod
 
 	@Override protected void initGUI() {
 		restrictionBiomes = new BiomeListField(mcreator, true);
+		ignoreBlocks = new MCItemListField(mcreator, ElementUtil::loadBlocks);

```

Comment:
Maybe it would make sense to list air and structure block by default if not editing mode?

Reply:
Maybe just structure block as it's the default value prior to this PR?

Reply:
Also ok

Reply:
✔️ Fixed

Reply:
I don't see the code that specifies default value. Where can I find it?

Reply:
Oops, forgot IDEA rejected my push in favor of remote changes.

Reply:
Should be visible now.